src/Frontend/Modules/Search/Actions/Index.php

```diff
@@ -147,16 +113,14 @@ private function getRealData(): void
             throw new NotFoundHttpException();
         }
 
-        if ($this->getContainer()->getParameter('kernel.debug')) {
+        if (!$this->getContainer()->getParameter('kernel.debug')) {
```

Comment:
I don't get what this line is doing here. Could you add a comment?

Reply:
woops, that was debug code :)
fixed in 80fc6ae80